src/registrar/models/domain_application.py

```diff
@@ -571,7 +571,12 @@ def _send_status_update_email(self, new_status, email_template, email_template_s
 
         if self.submitter is None or self.submitter.email is None:
             logger.warning(f"Cannot send {new_status} email, no submitter email address.")
-            return
+            return None
+
+        if do_fake_send_email:
+            logger.info(f"Email was not sent. Would send {new_status} email: {self.submitter.email}")
```

Comment:
Optional suggestion: Think it'd be good to explain why the email was not sent (similar to the above Email not sent message) so the user understands that the email not being sent is expected
` logger.warning(f"Cannot send {new_status} email, no submitter email address.")`

Reply:
Great suggestion, thank you